src/main/java/duke/exception/InvalidTaskNoException.java

```diff
@@ -0,0 +1,13 @@
+package duke.exception;
+
+/**
+ * Represents an exception thrown when an invalid task number is given. A subclass of DukeException.
+ */
+public class InvalidTaskNoException extends DukeException {
+    /**
+     * Constructor of the class `InvalidTaskNoException`.
+     */
+    public InvalidTaskNoException() {
+        super("☹ OOPS!!! The task number is invalid.");
+    }
+}
```

Comment:
I think it is really great that you have multiple types of `DukeException`! This makes understanding the exceptions thrown in this whole project more easily! 👍